.circleci/config.yml

```diff
@@ -256,6 +307,46 @@ jobs:
                 ci_build_num:$CIRCLE_BUILD_NUM \
                 commit_sha:$CIRCLE_SHA1
 
+  release-ios:
+    macos:
+      xcode: "9.2.0"
+    working_directory: /Users/distiller/project/ios
+    environment:
+      FL_OUTPUT_DIR: /Users/distiller/project/output
+    shell: /bin/bash --login -o pipefail
+    steps:
+      - checkout:
+          path: /Users/distiller/project
+      - run:
+          name: Set Ruby Version
+          command:  echo "ruby-2.4" > ~/.ruby-version
+      # Not using a workspace here as Node and Yarn versions
+      # differ between the macOS image and the Docker containers above.
+      - <<: *restore-gems-cache
+      - <<: *ruby-dependencies
+      - <<: *save-gems-cache
+      - <<: *get-cache-osx
+      - run:
+          name: Set npm version
+          command: |
+              npm i -g n
+              sudo n 9.3
+      - run:
+          name: Install dependencies
+          command: yarn install
+      - <<: *save-cache-osx
+      - run:
+          name: Generate .env
+          command: node ./scripts/generate-dotenv.js
+          working_directory: /Users/distiller/project
+      - run:
+          name: Set Fabric secret
+          command: echo -e "$FABRIC_API_KEY\n$FABRIC_BUILD_SECRET" >> ./PrideLondonApp/fabric.properties
+      - run:
+            command: |
+                bundle exec fastlane release \
```

Comment:
Separated TestFlight build and submit into it's own job so can be run in parallel